app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -72,6 +89,16 @@ class ProjectBoard extends React.Component {
           />
         </Column>
 
+        {
+          hasSearch(projectBoard) &&
```

Comment:
What do you think about creating a custom connected component that uses `Column` inside of if called `SearchResults`? Then you could render it here and use `hasSearch` only inside there, like this:

```js
const SearcResults = ({ isEnabled, searchResults }) => {
  if(!isEnabled) {
    return null;
  }

  return (<Column {/* props here */}>{ /* content here */ }</Column>);
}

const mapStateToProps = (state) =({
  isEnabled: hasSeach(state.projectBoard),
  searchResults: fromSearch(stories) // this is another selector that has fromScope(stories, 'search') inside of it
});

const mapDispatchToProps = {
  closeSearch
}

exports default connect(mapStateToProps, mapDispatchToProps)(SearchResults);
}